openid-4-verifiable-presentations-1_0.md

```diff
@@ -809,11 +809,11 @@ Note: Some of the processing rules of the Presentation Definition and the Presen
 
 # Wallet Invocation {#wallet-invocation}
 
-The Verifier has the choice of the following mechanisms to invoke a Wallet:
+The Verifier MAY use one of the following mechanisms to invoke a Wallet:
```

Comment:
let's keep this PR honest to its title and not introduce normative changes in the editorial PR

```suggestion
The Verifier can use one of the following mechanisms to invoke a Wallet:
```